CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [unreleased] yyyy-mm-dd
+
+### Dynamic Addressables Network Prefabs
+
+#### Added
+- Added Multiplayer Play Mode to the project (#271)
+
+#### Changed
+- Upgraded the project to Unity 6000.0.34f1 LTS (#271) Other changes include:
+  - Netcode for GameObjects upgraded from v1.7.1 to v2.2.0
+  - ParrelSync removed from the project
```

Comment:
small typos here: there seems to be an extra space in those lines

Reply:
At the trailing end?
If it's the ones at the start, they're meant to be there since the line above is suggesting that there's a list of changes with the colon.